migration/src/main/scala/akka/persistence/r2dbc/migration/MigrationTool.scala

```diff
@@ -283,6 +283,7 @@ class MigrationTool(system: ActorSystem[_]) {
       case Some(selectedSnapshot @ SelectedSnapshot(snapshotMetadata, _)) =>
         for {
           seqNr <- {
+            // TODO we could load the timestamp and tags from corresponding event, see R2dbcSnapshotStore.saveAsync
```

Comment:
In MigrationTool I don't think this is important, and that's why I used TODO instead of FIXME. Instead, when enabling the feature they should run the sql migration step as documented.

Reply:
I'll change the comment, and mention what I said instead. Not TODO.